x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/lib/calculate_from_based_on_metric.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Moment } from 'moment';
+import { SnapshotCustomMetricInput } from '../../../../../common/http_api';
+import {
+  InventoryItemType,
+  SnapshotMetricType,
+} from '../../../../../common/inventory_models/types';
+import { InventoryMetricConditions } from '../types';
+import { isRate } from './is_rate';
+import { findInventoryModel } from '../../../../../common/inventory_models';
+
+export const calcualteFromBasedOnMetric = (
```

Comment:
@simianhacker I see it's merged. I just put the comment as a reminder to change `calcualteFromBasedOnMetric` to `calculateFromBasedOnMetric`